examples/tokens/token_create_transaction_token_fee_schedule_key.py

```
@@ -145,7 +138,10 @@ def attempt_fee_update(client, token_id, fee_schedule_key, description):
 
 
 def main():
-    client, operator_id, operator_key = setup_client()
+
+    client = setup_client()
+    operator_id = client.operator_account_id
+    operator_key = client.operator_private_key
```

Ruff (0.14.10)

Local variable `operator_key` is assigned to but never used

Remove assignment to unused variable `operator_key`

(F841)